anni/src/subcommands/repo.rs

```diff
@@ -238,6 +246,64 @@ fn repo_get_vgmdb(options: &RepoGetVGMdb, manager: &RepositoryManager, get: &Rep
     Ok(())
 }
 
+#[derive(Parser, Debug, Clone)]
+pub struct RepoGetCue {
+    #[clap(short = 'p', long)]
+    path: String,
+    /// if metadata is not enough, search vgmdb with keyword
```

Comment:
Use `//` here. `///` would be parsed as help message by `clap-derive`. We maintain those messages in separatee i18n files.

Reply:
I'll move it to `anni.ftl`